pyodide-build/pyodide_build/buildall.py

```diff
@@ -476,11 +472,21 @@ def build_packages(packages_dir: Path, outputdir: Path, args) -> None:
     pkg_map = generate_dependency_graph(packages_dir, packages)
 
     build_from_graph(pkg_map, outputdir, args)
+    for pkg in pkg_map.values():
+        if isinstance(pkg, StdLibPackage):
+            pkg.file_name = pkg.name + ".tar"
+            continue
+        if pkg.shared_library:
+            pkg.file_name = f"{pkg.name}-{pkg.version}.zip"
+            continue
+        for file in (pkg.pkgdir / "dist").glob("*.whl"):
```

Comment:
That's a bit wide. Maybe let's at least put the `pkg.name.replace('-', '_')` and the version in the glob and make a case insensitive search. Then check that there is only one result? Or alternatively validate the path with https://packaging.pypa.io/en/latest/utils.html#packaging.utils.parse_wheel_filename

Reply:
How do I do case insensitive search?

Reply:
Maybe https://stackoverflow.com/a/8151371/1791279 but yeah it looks not obvious. Just do something to avoid the situation where multiple wheels end up being in that folder and we pick the wrong one.

Reply:
How about for now we add an assert that it only contains one `.whl` file? The options should either be a `.whl`, a `.whl` and a `.tar` with unvendored tests, or a `.tar` with a shared lib.

Reply:
I guess maybe if you update the version of the package or something, we will end up with two wheel files in the `dist` directory. Probably we'd want to pick the right wheel in that case rather than crashing. Or we could empty the `dist` directory before putting the new stuff into it...

Reply:
Okay, now I explicitly assert that there is only one wheel in the dist directory. I think this makes it less error prone.